scripts/post-install.sh

```diff
@@ -24,10 +24,8 @@ function install_chkconfig {
     chkconfig --add telegraf
 }
 
-id telegraf &>/dev/null
-if [[ $? -ne 0 ]]; then
-    grep "^telegraf:" /etc/group &>/dev/null
-    if [[ $? -ne 0 ]]; then
+if ! id telegraf &>/dev/null; then
+    if grep "^telegraf:" /etc/group &>/dev/null; then
```

Comment:
Ohhh sure....Yeah there's a `!` missing in front of the grep like one line above. will fix this immediately!